fr.sgued.ten-forward.json

```diff
@@ -0,0 +1,46 @@
+{
+  "app-id": "fr.sgued.ten-forward",
```

Comment:
This should be `fr.sgued.ten_forward` https://freedesktop.org/software/appstream/docs/chap-Metadata.html#tag-id-generic

Reply:
Should i change it everywhere?

Reply:
Done

Reply:
there was nothing wrong. the last component (application name) doesn't have the restrictions.

Reply:
The spec seems to say any underscores in the id 

> ...replace any hyphens in the ID

Reply:
> Well it does cause issues: [Relm4/relm4-template#26](https://github.com/Relm4/relm4-template/issues/26)

That's because it's no longer the last component.

Reply:
> The spec seems to say any underscores in the id
> 
> > ...replace any hyphens in the ID

Not the flatpak documentation. But I guess when it is a hodgepodge of stubborn spec where people disagree... I mean they are the same the say to add `.desktop`.


Anyway, it is valid and it works, but I just wanted to point out it wasn't necessary.

Reply:
Hmm yea, I didn't notice that they differed here

Reply:
I opened an issue for clarification https://github.com/ximion/appstream/issues/547